RadixWallet/Core/SharedModels/Entities/EntitiesControlledByFactorSource.swift

```diff
@@ -24,6 +26,55 @@ public struct EntitiesControlledByFactorSource: Sendable, Hashable, Identifiable
 }
 
 extension EntitiesControlledByFactorSource {
+	public struct PerCurve: Equatable, Sendable {
+		public struct ID: Sendable, Hashable {
+			public let factorSourceID: FactorSource.ID.FromHash
+			public let isOlympia: Bool
+		}
+
+		public let id: ID
+		public let accounts: NonEmptyAccounts
+		public let hiddenAccounts: NonEmptyAccounts?
+	}
+
+	public var olympia: PerCurve? {
```

Comment:
I find `PerCurve` a confusing name. Is it not just a list of accounts of a single, given curve?

Reply:
`AccountsOnSameCurve`?

Reply:
hmm

Reply:
sure

Reply:
`AccountsControlledByKeysOnSameCurve`